lib/onebox/engine/amazon_onebox.rb

```diff
@@ -38,15 +44,15 @@ def image
 
       def data
         result = { link: link,
-                   title: raw.css("h1").inner_text,
+                   title: raw.css("title").inner_text,
```

Comment:
Updated this because we can consistently get the `title` even if all else fails. :smile_cat: